test/testfixture/make_functions.go

```diff
@@ -515,3 +516,35 @@ func makeTrackers(fxt *TestFixture) error {
 	}
 	return nil
 }
+
+func makeQueries(fxt *TestFixture) error {
+	if fxt.info[kindQueries] == nil {
+		return nil
+	}
+	fxt.Queries = make([]*query.Query, fxt.info[kindQueries].numInstances)
+	queryRrepo := query.NewQueryRepository(fxt.db)
+
+	for i := range fxt.Queries {
+		fxt.Queries[i] = &query.Query{
+			Title:   testsupport.CreateRandomValidTestName("label "),
```

Comment:
Can we use a better word that `label ` here?

Reply:
haha `copy-paste` did that, not me :wink: 
Fixed in https://github.com/fabric8-services/fabric8-wit/pull/1824/commits/2c7f2d8b82dfd68ccfa159a7e6373bfe991266c4
thanks!